src/full/Agda/TypeChecking/Serialise/Base.hs

```diff
@@ -44,16 +45,16 @@ type Node = [Int32]
 
 -- | The type of hashtables used in this module.
 --
--- A very limited amount of testing indicates that 'H.CuckooHashTable'
--- is somewhat slower than 'H.BasicHashTable', and that
--- 'H.LinearHashTable' and the hashtables from "Data.Hashtable" are
--- much slower.
+-- A very limited amount of testing indicates that
+-- 'Data.HashTable.IO.CuckooHashTable' is somewhat slower than
+-- 'Data.HashTable.IO.BasicHashTable', and that
+-- 'Data.HashTable.IO.LinearHashTable' and the hashtables from
+-- "Data.Hashtable" are much slower. However, the following hashtable
+-- from "Data.Vector.Hashtables" seems to be quite a bit faster than
+-- 'Data.HashTable.IO.BasicHashTable'.
```

Comment:
Please include benchmark data or a link to the study.

Reply:
I can refer to issue #5966 (see https://github.com/agda/agda/issues/5966#issuecomment-1161401330).